packages/components/src/Typeahead/Typeahead.component.renderers.js

```diff
@@ -258,7 +258,7 @@ export function renderItem(item, { value, ...rest }) {
 				[theme.selected]: value === title,
 			})}
 			title={title}
-			data-feature={item['data-feature'] || rest['data-feature']}
+			data-feature={item['data-feature'] || rest['data-feature'] || item.value}
```

Comment:
With that, every datalist (even the ones that we don't want to get pendo stats) will have a data-feature, with only the item value. This will pollute the pendo data.
It's better to let the dev explicitly set the data-feature. You already set the item api to set a data-feature by item with `item['data-feature']`. Why having this `item.value` ?

Reply:
I agree with @jsomsanith-tlnd, our tag values are not enough specific today to authorize this kind of fallback, you can easily mess the tracking up!

Reply:
for what I know, pendo tracking is opt-in so if the rule to target the click is not create on pendo platform, it won't appear

`item['data-feature']` need the backend to send and store that data which is related to tck, otherwise mapping the collection somewhere like Datalist.component ```items={
                        this.state.suggestions &&
                        this.state.suggestions.map(suggestion => ({
                            ...suggestion,
                            'data-feature': suggestion.value,
                        }))
                    }``` that will end up being available in pendo as current fallback solution but will need a realloc that will slow down apps